src/components/send/SendHeader.js

```diff
@@ -133,7 +133,7 @@ export default function SendHeader({
 
   return (
     <Fragment>
-      {ios && <SheetHandle />}
+      <SheetHandle />
```

Comment:
I already fixed this. Why am I seeing it again?

Reply:
you both solved in different ways so when I pulled I had 2 handles, this seemed like the best solution and I tested on both OS

Reply:
if it works, it works